src/store/getters/state-getters.ts

```diff
@@ -297,11 +297,5 @@ export const getCertifyState = (state: StateIF): CertifyIF => {
 
 /** The Completing Party's email address. */
 export const getCompletingPartyEmail = (state: StateIF): string => {
-  const completingParty =
-    getAddPeopleAndRoleStep(state).orgPeople.find(person => {
-      return Boolean(person.roles?.some(role => {
-        return (role.roleType === RoleTypes.COMPLETING_PARTY)
-      }))
-    })
-  return (completingParty?.officer?.email || null)
+  return (state.stateModel.tombstone.userEmail)
```

Comment:
Does this business logic change work for BEN IAs too?

Also, you can use `getTombstone(state).userEmail` now :)

Reply:
Yea! Works for both :)
Originally was only applying the email once the completing party had been filled out. 
Although we retrieved the email right up front and always had it.